src/schemas/player.json

```diff
@@ -0,0 +1,50 @@
+{
+  "title": "Player",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string"
+    },
+    "role": {
```

Comment:
Might be better as 'position' if we're being picky, but role works too

Reply:
Updated